### PR TITLE
Update readme structure/texts (including link to DE wiki page)

### LINKED ISSUE #65: Link to instructions
Found the game page via youtube, registered, was lost. 

Link to https://wiki.cryptic-game.net/books/spielanleitung from readme.

### PR DESCRIPTION
**Pull request by cadamini**
**Description**
Improves texts in readme and add link to DE instructions wiki page

**Issue**
Closes https://github.com/cryptic-game/cryptic/issues/65

### CODE REVIEW

**Comment by Tristan-H11** [comment]
The wiki page you linked is about how to play the game. Maybe the link would be better off a section lower in "I want to play!" instead of under the question, what the game itself is at all. <3

**Comment by cadamini** [comment]
@Tristan-H11 Updated texts a bit more and moved the link. Feel free to close this PR if this was too much

File: README.md [hunk 1]
```diff
@@ -4,45 +4,39 @@
...
 
-## What is it?!
+## What is it?
 
-Cryptic is an deep strategy game.
-You start with only one virtual computer ingame and a few crypto coins.
-The goal of cryptic is to hack other players and think creative like real hackers.
-You must think hard about your actions before attacking another player and learn the basics of the game.
-But all in all it's not so hard to start!
+Cryptic is an deep strategy game. The goal of this deep strategy game is to hack other players and think creative like real hackers. You only start with a virtual computer and a few crypto coins. You must think carefully about your actions before attacking another player and learn the basics of the game. But it's not hard to get started.
```
Comment (Tristan-H11) [inline README.md]:
```suggestion
Cryptic is a deep strategy game. The goal of this deep strategy game is to hack other players and think creatively like real hackers. You only start with a virtual computer and a few crypto coins. You must think carefully about your actions before attacking another player and learn the basics of the game. But it's not hard to get started.  
```

File: README.md [hunk 2]
```diff
@@ -4,45 +4,39 @@
...
 
-## I want to help!
+## Support our team!
 
-No matter what skills you have... ALL ARE HELPFUL!
-From design to development. For more information talk to us on discord or write us an email.
+Contribute to Cryptic to become part of our team.
 
-## Organisation
+Cryptic is driven by a not profit-oriented team that maintains the game, adds new features and releases new versions.
```
Comment (Tristan-H11) [inline README.md]:
```suggestion
Cryptic is driven by a non-profit-oriented team that maintains the game, adds new features, and releases new versions.
```

**Review by Tristan-H11 — CHANGES_REQUESTED** [review]
If you don't mind, please implement these changes, so we can merge the PR. :)

**Review by Tristan-H11 — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -4,45 +4,39 @@
...
-The main server is hosted on wss://ws.cryptic-game.net/, but you can [host your own](https://github.com/cryptic-game/server) for family and friends.
+The main server is hosted on wss://ws.cryptic-game.net/ (and automatically used when you play in the browser), but you can [host your own](https://github.com/cryptic-game/server) for family and friends.
+
+### 3. Create an account and have fun
 
-### 3. Have FUN!
+[Create a Cryptic account to play](https://play.cryptic-game.net/signup) and become the strongest hacker of Cryptic.  
 
-At the end you have to create an account. After that you can become the strongest hacker of cryptic.
+Refer to our [wiki](https://wiki.cryptic-game.net/books/spielanleitung) (German) for instructions or visit our Youtube channel for Let's Play videos.
```
Comment (cadamini) [inline README.md]:
```suggestion
Refer to our wiki for [German](https://wiki.cryptic-game.net/books/spielanleitung) or [English](https://wiki.cryptic-game.net/books/game-manual) instructions or visit our Youtube channel for a Let's Play video.
```

**Review by Tristan-H11 — APPROVED** [review]

**Review by Defelo — APPROVED** [review]

**Pull request merged**